mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_FileManager.java

```diff
@@ -103,16 +103,18 @@ public static void copyStream(final InputStream inStream, final OutputStream out
         map.put("description", "");
         map.put("category", "Misc");
         final BufferedReader reader = new BufferedReader(new StringReader(header));
-        String headerLine;
         try {
+            String headerLine;
             while ((headerLine = reader.readLine()) != null) {
                 if (headerLine.matches("//.*@.*")) {
```

Comment:
```suggestion
                if (headerLine.matches("^//\s*@.*")) {
```
Shouldn't it be like this?

Reply:
Actually I would prefer Scanner here, I would send a new commit soon